xbmc/pvr/epg/EpgSearchFilter.cpp

```diff
@@ -114,7 +117,7 @@ bool CPVREpgSearchFilter::MatchDuration(const CPVREpgInfoTagPtr &tag) const
 
 bool CPVREpgSearchFilter::MatchStartAndEndTimes(const CPVREpgInfoTagPtr &tag) const
 {
-  return (tag->StartAsLocalTime() >= m_startDateTime && tag->EndAsLocalTime() <= m_endDateTime);
+  return (tag->StartAsLocalTime() >= GetStartDateTime() && tag->EndAsLocalTime() <= GetEndDateTime());
```

Comment:
Did you check if `m_startDateTime` and `m_endDateTime` are used anywhere else still?